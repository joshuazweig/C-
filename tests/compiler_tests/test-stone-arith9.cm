int main () {
    stone s1;
    stone s2;
    stone s3;

    s1 = "4261379284712349123746192345";
    s2 = "8";
    s3 = s1 ^ s2;

    print_stone(s3);
}
