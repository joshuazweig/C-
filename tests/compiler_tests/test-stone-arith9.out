4D035B624E41A535845CB3ADDF265B2AAED496093FCDE28BCA87C580A0B03ECEBAB5A7E0DF4154AA9AC101B9872178905F7D51FB5318A74C046CD26DFEAAEB30751F5086A129E855174CD19595E35C66D424377AA33170A144CE3DC1
